### PR TITLE
Implement cookie expire time - forgetConsentGiven

### LINKED ISSUE #19530: Allow custom cookie expiry date for `optUserOut()` & `forgetUserOptOut()` functions
It would be beneficial to overwrite the default cookie expiry date for the `optUserOut()` and 'forgetUserOptOut()', especially since you can also set custom values on most other functions which set cookies for tracking.

## Summary

Adding an additional parameter to the `optUserOut()` & `forgetUserOptOut()` functions.

**Example:**

```
_paq.push(['optUserOut', 2160]);
```

would set the `mtm_consent_removed` to an expiry date in 90 days.

## Your Environment

* Matomo Version: 4.10.1
* PHP Version: 7.3.28

### PR DESCRIPTION
**Pull request by futureweb**
### Description:

Implement cookie expire time - forgetConsentGiven Feature Request #19530

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [x] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [x] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [x] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by sgiehl** [comment]
Hi @futureweb Thanks for creating this PR. 
Changes to piwik.js require the minified tracker files to be rebuilt. See https://github.com/matomo-org/matomo/blob/4.x-dev/js/README.md#L32 for instructions how to update such files.

**Comment by futureweb** [comment]
> Hi @futureweb Thanks for creating this PR. Changes to piwik.js require the minified tracker files to be rebuilt. See https://github.com/matomo-org/matomo/blob/4.x-dev/js/README.md#L32 for instructions how to update such files.

Hi @sgiehl - thx for the info, was my first commit within the .js scope ... ;-)
Have updated the minified tracker files as advised.

**Comment by sgiehl** [comment]
@futureweb Thanks for that. Seems you missed to also update the file `/piwik.js`

**Comment by futureweb** [comment]
@sgiehl - darn ... never noticed there are already so much redundant versions of the JS since the Name-Switch ... ^^ ... hope I got em all now 😅

File: js/piwik.js [hunk 1]
```diff
@@ -7223,11 +7223,15 @@ if (typeof window.Matomo !== 'object') {
              * consent will be enforced. You may call this method if the user removes consent manually, or if you
              * want to re-ask for consent after a specific time period.
              */
-            this.forgetConsentGiven = function () {
-                var thirtyYears = 30 * 365 * 24 * 60 * 60 * 1000;
+            this.forgetConsentGiven = function (hoursToExpire) {
```
Comment (sgiehl) [inline js/piwik.js:7231]: would be nice to have the new param described in the doc block above.

Comment (futureweb) [reply js/piwik.js:7231]: @sgiehl of course you are right! added! :-)

**Review by sgiehl — APPROVED** [review]
Looks good to me now 🎉 
Thanks for the contribution @futureweb

**Pull request merged**